proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -36,15 +36,15 @@ service Query {
   rpc QueryValidatorConsumerAddr(QueryValidatorConsumerAddrRequest)
       returns (QueryValidatorConsumerAddrResponse) {
     option (google.api.http).get =
-        "/interchain_security/ccv/provider/validator_consumer_addr";
+        "/interchain_security/ccv/provider/{consumer_id}/{provider_address}";
```

Comment:
We had issues with this on the last PR about queries. The queries were overlapping on accident and one of the fixed strings could have been interpreted as a dynamic string.
```suggestion
        "/interchain_security/ccv/provider/validator_consumer_addr/{consumer_id}/{provider_address}";
```